src/spikeinterface/core/base.py

```diff
@@ -557,7 +560,9 @@ def dump(self, file_path: Union[str, Path], relative_to=None, folder_metadata=No
         else:
             raise ValueError("Dump: file must .json or .pkl")
 
-    def dump_to_json(self, file_path: Union[str, Path, None] = None, relative_to=None, folder_metadata=None) -> None:
+    def dump_to_json(
+        self, file_path: Union[str, Path, None] = None, relative_to=None, folder_metadata=None, recursive=False
+    ) -> None:
```

Comment:
I do not understand. I think that json should be always reccursive  no ?
how could it be not reccurssive ?